File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/GoodsService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.group.Goods;
import com.pinyougou.pojo.TbGoods;
import com.pinyougou.utils.PageResoult;
import com.pinyougou.utils.Resoult;

public interface GoodsService {
    Resoult updateGoods(Goods goods);

    Resoult addGoods(Goods goods);

    PageResoult<TbGoods> findAll();

    PageResoult<TbGoods> searchByCondition(int currentPage, int pageSize, TbGoods goods);

    Resoult deleteGoods(long[] items);

    Resoult review(String auditStatus, long[] items);

    Goods findOne(long id);

    Resoult deleteGoodsByManager(long[] items);

    Resoult superMarket(long id, String market);
}

File: pinyougou-manager-web/src/main/webapp/service/goodsService.js
app.service("goodsService",function ($http) {

    //查询全部商品
    this.findAll = function () {
        return $http.get("/goods/findAll.do");
    }

    //条件查询带分页
    this.searchByCondition = function (currentPage,itemsPerPage,seachEntity) {
        return $http.post("/goods/searchByCondition.do?currentPage=" + currentPage + "&pageSize=" + itemsPerPage,seachEntity);
    }

    this.deleteGoods=function (selectedItems) {
        return $http.post("/goods/deleteGoods.do",selectedItems);
    }

    this.review=function (auditStatus,selectedItems) {
        return $http.post("/goods/review.do?auditStatus=" + auditStatus,selectedItems);
    }
})